typescript-json-schema.ts

```diff
@@ -63,14 +63,14 @@ export type Definition = {
     type?: string | string[],
     definitions?: {[key: string]: any},
     format?: string,
-    items?: Definition,
+    items?: Definition | Array<Definition>,
```

Comment:
Let's make this consistent and use `[]` instead of `Array`. Can you also correct the defs below?